build.gradle

```diff
@@ -434,6 +434,14 @@ configurations {
             force "com.github.luben:zstd-jni:${versions.zstd}"
             force "org.xerial.snappy:snappy-java:1.1.10.3"
             force "com.google.guava:guava:${guava_version}"
+
+            // TODO: Seems like this should be removable
```

Comment:
Need to resolve this comment

Reply:
Should we remove this btw? Or is there any reason for forcing this? I do see that we have transitive dependencies here: https://github.com/opensearch-project/security/blob/ba3a7014d1c9c19e07f5087742bcf4ab59e04a88/build.gradle#L494

Reply:
I'll update it in a fast follow PR